src/index.tsx

```diff
@@ -4,13 +4,13 @@ import NewsFeed from './NewsFeed/NewsFeed';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-	<React.StrictMode>
-		<main id="main" className="container">
-			<h1>Main page title</h1>
-			<NewsFeed />
-		</main>
-	</React.StrictMode>,
-	document.getElementById('root')
+  <React.StrictMode>
```

Comment:
For cleaner code you can assign this into an <App> component that you can pass later to the render function